app/controllers/checkout_forms/addresses_form.rb

```diff
@@ -0,0 +1,13 @@
+class AddressesForm < Rectify::Form
+  attribute :billing_address, BillingAddressForm
+  attribute :shipping_address, ShippingAddressForm
+
+  def save
+    if valid?
```

Comment:
you can use a guard statement here to make a code cleaner. Also, no need for explicitly returning true - if model.save is successful it will return true by itself:

```
def save
  return if invalid?
  billing_address.save && shipping_address.save
end
```